pagopa/io-dev-api-server
# [IOCOM-739] Add the `has_remote_content` property to messages with remote content

## Short description
This PR adds the `has_remote_content` property to messages with remote content.

## List of changes proposed in this pull request
- updated specs
- updated `withRemoteContent` in order to return the `has_remote_content` property

## How to test
- Run the server with the configuration that returns a message with remote content
```
{
   count: 1,
   template: {
      subjectWordCount: 5,
      hasRemoteContent: true,
      attachmentCount: 5
   }
},
  ```
- Verify that the endpoint `/messages/:id` returns the `has_remote_content`
